demo/src/app/components/alert/demos/closeable/alert-closeable.html

```diff
@@ -1,6 +1,6 @@
-<p *ngFor="let alert of alerts">
-  <ngb-alert [type]="alert.type" (close)="closeAlert(alert)">{{ alert.message }}</ngb-alert>
-</p>
+<ngb-alert [type]="alert.type" (close)="closeAlert(alert)" *ngFor="let alert of alerts">
```

Comment:
Ah, I struggled when trying to do this one myself, good shout on moving the ngFor onto the alert element itself to get the leave animation to trigger properly. Maybe this should be documented?

Reply:
I'm going to discuss this one with Matias and see what would be the best way of progressing.

Reply:
Could you nudge him on https://github.com/angular/angular/issues/18749 as well, as we'll need that for the collapse animation (as it's not currently possible to add animations via directive metadata).